Automattic/pocket-casts-android
# Enable Catalan language

## Description

This enables the Catalan language.

You can see that app is more or less fully translated here: https://translate.wordpress.com/projects/pocket-casts/android/ca/default

Closes #3022

## Testing Instructions

1. Change your system language to Catalan.
2. Verify that the app supports it.

## Checklist
